index.js

```diff
@@ -17,12 +14,12 @@ const app = express();
 
 app.use(cors());
 
+
 // Normal express config defaults
 app.use(require("morgan")("dev"));
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
-app.use(require("method-override")());
 app.use(express.static(__dirname + "/public"));
```

Comment:
Do we really need any static files, since this is just the backend? 🤔 
@chekwas88